src/components/dropdownList/dropdownList.tsx

```diff
@@ -0,0 +1,28 @@
+import { NavLink } from "react-router-dom";
+import { CategoryInfo, categoryInfos } from "@/categoriesInfo";
+import styles from "./dropdownList.module.scss";
```

Comment:
where are the styles?

Reply:
this file is a mistake copy. original DropdownList with styles is located in /elements folder. This incorrect copy is removed